### ISSUE TITLE
Can't replicate Text BERT Acc/AUROC Even with pretrained model

### ISSUE THREAD

**Issue by hivestrung** [facebookresearch/mmf#926]
Hi,

With the test set labels released on Hateful Memes new webpage, I ran evaluation metrics on test_seen:

```
mmf_run config=projects/hateful_memes/configs/unimodal/bert.yaml model=unimodal_text dataset=hateful_memes run_type=test checkpoint.resume_zoo=unimodal_text.hateful_memes.bert checkpoint.resume_pretrained=False dataset_config.hateful_memes.annotations.val[0]=hateful_memes/defaults/annotations/dev_seen.jsonl dataset_config.hateful_memes.annotations.test[0]=hateful_memes/defaults/annotations/test_seen.jsonl
```
The results I got:
![image](https://user-images.githubusercontent.com/27841209/117134447-82270c80-ad9d-11eb-93fb-7e903a5ab769.png)

The results I expect (taken from latest version on arxiv)
<img width="666" alt="image" src="https://user-images.githubusercontent.com/27841209/117134617-b3074180-ad9d-11eb-9d16-6e30e8aba2f6.png">

There might be a problem here, Text BERT as reported in the paper is way too good. It even beats Concat BERT and is on par with MMBT-Grid. 

tagging @apsdehal

**Comment by vedanuj** [comment]
The result reported is using the newer version of train set tested on test seen set. The checkpoint released are not on newer version of train set. Also the results are average of multiple runs.

**Comment by hivestrung** [comment]
@vedanuj thanks so much. Another question, the result reported for validation set - also val_seen, I am guessing?
